runtime/drive_handler_test.go

```diff
@@ -32,8 +32,7 @@ import (
 )
 
 func TestStubDriveHandler(t *testing.T) {
-	const tempPath = "test"
-	err := os.Mkdir(tempPath, os.ModePerm)
+	tempPath, err := ioutil.TempDir("./", "TestStubDriveHandler")
 	assert.NoError(t, err)
```

Comment:
nit: this should probably be `require.NoError` instead of `assert` because the rest of the test is destined to fail if the temp dir fails to create.

Reply:
Fixed